neurons/validator.py

```diff
@@ -389,7 +391,8 @@ def main(self):
                 self.challengeperiod_manager.refresh(current_time=current_time)
                 self.weight_setter.set_weights(current_time=current_time)
                 self.elimination_manager.process_eliminations()
-                self.position_syncer.sync_positions_with_cooldown(self.auto_sync)
+                if not self.is_mothership:
```

Comment:
better to simply change the initialization of self.auto_sync so we gate in a single location